content/movie-review-pt2.md

```diff
@@ -10,8 +10,9 @@
 
 # TL;DR
 
-- Creating an account requires that we calculate the space and rent to allocate for a new account
-- A Program Derived Address (PDA) is derived from a program Id and an optional list of seeds
+- Program state is stored in external accounts rather than in the program itself
+- A Program Derived Address (PDA) is derived from a program Id and an optional list of seeds and subsequently used as the address for a storage account
```

Comment:
sugg: add a comma, or make a new sentence... too many "and"s, e.g. 

A Program Derived Address (PDA) is derived from a program Id and an optional list of seeds. Once derived, PDAs are subsequently used as the address for a storage account.

or,

A Program Derived Address (PDA) is derived from a program Id and an optional list of seeds, and is subsequently used as the address for a storage account